notesapi/v1/views.py

```diff
@@ -55,11 +56,12 @@ def get(self, *args, **kwargs):  # pylint: disable=unused-argument
         """
         Get a list of all annotations.
         """
-        self.kwargs['query'] = self.request.QUERY_PARAMS.dict()
-
-        annotations = Annotation.search(**kwargs)
+        params = self.request.QUERY_PARAMS.dict()
+        results = NoteMappingType.process_result(
```

Comment:
> sorry i only just noticed this, but it looks like List and Detail get methods are fetching notes from ES instead of the database. I don't think we want this - my assumption would have been that we used the database as a primary source of truth. Let me know if I'm misunderstanding what's going on here.

Addressed. Fetching notes from ES was done as it was assumption to use mysql only when storing/udating/deleting notes. Now List and Detail ``get`` methods use database.